aws/credentials/credentials_bench_test.go

```diff
@@ -55,7 +55,7 @@ func BenchmarkCredentials_Get_Expire(b *testing.B) {
 						for j := 0; j < b.N; j++ {
 							v, err := creds.Get()
 							if err != nil {
-								b.Fatalf("expect no error %v, %v", v, err)
+								b.Errorf("expect no error %v, %v", v, err)
```

Comment:
is this intentional change?

Reply:
yeah `Fatalf`s are not valid in goroutines, the vet catches these.